### PR TITLE
CP-32847: Support manually block/unblock path

### PR DESCRIPTION
**Pull request by DeliZhangX**
Signed-off-by: Deli Zhang <deli.zhang@citrix.com>

### CODE REVIEW

**Comment by DeliZhangX** [comment]
Jobs passed:
2805244, 2805245, 2805246 (one subtest failed which is known issue), 2805248, 2805251~2805253, 2805257~2805259, 2805261~2805265, 2805271;
2805403~2805410, 2805412

File: src/XenCert/StorageHandler.py [hunk 1]
```diff
@@ -88,11 +89,12 @@ def run(self):
                 XenCertPrint("Could not write through the allocated disk space on test disk, please check the storage configuration manually. Exception: %s" % str(e))
 
 class WaitForFailover(Thread):
-    def __init__(self, session, scsiid, activePaths, noOfPaths):
+    def __init__(self, session, scsiid, activePaths, noOfPaths, checkFunc):
```
Comment (29217321) [inline src/XenCert/StorageHandler.py]: you may need to add a default value null for this new added parameter for backward compatibility?

Comment (DeliZhangX) [reply src/XenCert/StorageHandler.py]: Needless I think, as there exist only one place using this class and it is changed correspondingly

File: src/XenCert/StorageHandler.py [hunk 2]
```diff
@@ -652,7 +670,19 @@ def BlockUnblockPaths(self, blockOrUnblock, script, noOfPaths, passthrough):
...
             return stdout
         except Exception, e:            
-            raise e        
+            raise e
+
+    def WaitManualBlockUnblockPaths(self):
+        try:
+            cmd = [self.storage_conf['pathHandlerUtil']]
+            (rc, stdout, stderr) = util.doexec(cmd, '')
+            XenCertPrint(
```
Comment (minli1) [inline src/XenCert/StorageHandler.py]: Seems this function is for block/unblock the paths.
From the logs is a little bit confused:
`The path manually block/unblock utility returned ... `  --> `The manually block/unblock path utility returned ...`

**Pull request merged**